cypher-shell/src/main/java/org/neo4j/shell/prettyprint/PrettyPrinter.java

```diff
@@ -1,24 +1,24 @@
 package org.neo4j.shell.prettyprint;
 
+import org.neo4j.driver.internal.types.TypeRepresentation;
 import org.neo4j.driver.v1.Record;
 import org.neo4j.driver.v1.Value;
-import org.neo4j.driver.v1.exceptions.value.Uncoercible;
 import org.neo4j.driver.v1.types.Node;
 import org.neo4j.driver.v1.types.Path;
 import org.neo4j.driver.v1.types.Relationship;
 import org.neo4j.shell.cli.Format;
 import org.neo4j.shell.state.BoltResult;
 
 import javax.annotation.Nonnull;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  * Print the result from neo4j in a intelligible fashion.
  */
 public class PrettyPrinter {
 
+    public static final String COMMA_SEPARATOR = ",";
```

Comment:
What do you think about adding a space? eg: `", "`

It seems inconsistent in the printout atm where there are spaces after colons, and after some commas, but not all.